### PR TITLE
Issue #294: Monitoring Remote Linux

### PR DESCRIPTION
**Pull request by iguitton**
Wrote the use case

### CODE REVIEW

**Comment by iguitton** [comment]
@NassimBtk , @bertysentry : your feedback is welcome!

File: metricshub-doc/src/site/markdown/usecases/remote-linux.md [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+
+1. In the `config/metricshub.yaml` file, we configure the monitoring on a Linux machine through `SSH`: 
+
+    ```yaml
+          dev-nvidia-01:
+            attributes:
+              host.name: dev-nvidia-01
+              host.type: linux
+    ```
+2. Then, we configure the SSL protocol
```
Comment (NassimBtk) [inline metricshub-doc/src/site/markdown/usecases/remote-linux.md:29]: Typo, `SSL` instead of `SSH` 😉

**Review by NassimBtk — CHANGES_REQUESTED** [review]
A small update and we are good 😉.

File: metricshub-doc/src/site/markdown/usecases/remote-linux.md (hunk 1 above)
Comment (iguitton) [reply metricshub-doc/src/site/markdown/usecases/remote-linux.md]: Good catch @NassimBtk . Typo is now fixed :)

**Review by NassimBtk — APPROVED** [review]

**Pull request merged**